### ISSUE TITLE
[8.0.0-svg-PR#6593] colors order on fixed borders

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#6627]
### Description
Found during testing https://github.com/handsontable/handsontable/pull/6593.
This testing stage of 8.0.0-SVG applies to two PRs - #6593 (bug) and #6594 (fixed). 

On the fixed `customBorder`, which is hardened in the code and corresponds to this border e.g. `custom border-bottom` for the `fixedRowsTop bottom border` 
 color order is displayed not according to the order specified in the requirements. 

Lighter applies to darker at the same thicknesses. Yellow overlay black.
This can be seen after the event, mouse click or scrolling. 
This does not appear on #6594.

#### Code:
```

customBorders.push({
                row: i,
                col: j,
                top: {
                    width: 3,
                    color: 'black',
                },
                left: {
                    width: 3,
                    color: 'black',
                  },
                  bottom: {
                    width: 3,
                    color: 'yellow',
                  },
                  right: {
                    width: 3,
                    color: 'pink',
                  }
            })
```
#### Case 1 - after clicking by mouse
Left is #6593, right is #6594 
![image](https://user-images.githubusercontent.com/30186818/72070508-0f616a80-32ea-11ea-9fd2-8c271e2cf006.png)
![Screenshot 2020-01-09 at 14 15 52](https://user-images.githubusercontent.com/30186818/72070842-c2ca5f00-32ea-11ea-99e3-efc23c28c18f.png)
![Screenshot 2020-01-09 at 14 15 25](https://user-images.githubusercontent.com/30186818/72070838-c1009b80-32ea-11ea-9fa9-71b18b1a2299.png)

#### Case 2 
 scrolling horizontal with fixedColumnsLeft/frozen columns
#### Case 3 
scrolling vertical with fixedRowsTop and fixedRowsBottom (maybe somebody wants use rotated table, IMHO we should fix all fixedRows)
![6593-chrome-stage2](https://user-images.githubusercontent.com/30186818/72070910-ec838600-32ea-11ea-87c1-0a0065aec6a1.gif)

### Your environment
* Handsontable version: 8.0.0-svg-PR#6593

**Comment by warpech** [comment]
PRs #6593 and #6594 were merged into the main SVG borders PR #6157. In the current tip of the branch (`7610c6aca7e21069a23db3b9869c08922c8e0099`) I can't reproduce the problem. Maybe it was fixed with other fixes?

@aninde please confirm if any of the cases 1-3 is still valid with the current PR #6157.

**Comment by aninde** [comment]
It is true. this issue cannot be repeated on current  #6157. It only appeared on [#PR6593](https://github.com/handsontable/handsontable/pull/6593), maybe during merging with working a [PR#6594](https://github.com/handsontable/handsontable/pull/6594) a fix was created.  we can close this issue because there is no Pull request anymore it appeared.
